benchmark/README.md

```diff
@@ -75,6 +77,9 @@ TPCH timing results is written to stdout in the following form: `TPCH_Result,<la
     <number of iterations> \
     <true for sql tests, false for functional tests>
     ```
+In order to run with Python 3.5 (the default is 2.7) you will need to do the following on all the nodes in the cluster:

```

Comment:
Please provide the instructions on how to do this.

Reply:
Done